app/src/main/java/com/github/se/studybuddies/ui/video_call/StreamVideoHelper.kt

```diff
@@ -0,0 +1,86 @@
+package com.github.se.studybuddies.ui.video_call
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.Log
+import com.github.se.studybuddies.database.DatabaseConnection
+import com.google.firebase.auth.FirebaseAuth
+import io.getstream.log.Priority
+import io.getstream.video.android.core.StreamVideo
+import io.getstream.video.android.core.StreamVideoBuilder
+import io.getstream.video.android.core.logging.LoggingLevel
+import io.getstream.video.android.model.User
+import kotlinx.coroutines.flow.MutableStateFlow
+
+enum class InitializedState {
+  NOT_STARTED,
+  RUNNING,
+  FINISHED,
+  FAILED
+}
+
+@SuppressLint("StaticFieldLeak")
+object StreamVideoInitHelper {
+  private lateinit var auth: FirebaseAuth
+
+  private var isInitialising = false
+  private lateinit var context: Context
+  private val _initState = MutableStateFlow(InitializedState.NOT_STARTED)
+
+  fun init(appContext: Context) {
+    context = appContext.applicationContext
+  }
+
+  suspend fun reloadSdk() {
+    StreamVideo.removeClient()
+    loadSdk()
+  }
+
+  /** A helper function that will initialise the [StreamVideo] SDK. */
+  suspend fun loadSdk() {
+    if (StreamVideo.isInstalled) {
+      _initState.value = InitializedState.FINISHED
+      Log.w("StreamVideoInitHelper", "[initStreamVideo] StreamVideo is already initialised.")
```

Comment:
Something very small, but *very* often done in android => you can extract the string `StreamVideoInitHelper` for all logs, call it `TAG` and define it as a field in the class above

```kotlin
val TAG: String = "StreamVideoInitHelper"
```